_posts/2015-07-21-the-epages-selenium-framework.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "The ePages Selenium Framework"
+date: "2015-07-21 17:06:00"
+icon: ship
+tags: selenium, test, continuous delivery
+categories: tech-stories
+authors: ["Jan M.", "Martin K.", "Kay A."]
+---
+
+With the help of Selenium we create and run automated test scripts for
+the user interface of ePages. In order to make it quick and easy for
+the developers to write these tests we have built the ePages
+Selenium Framework.
+
+# Browser automation with Selenium
+
+[Selenium] is an open source project which provides a couple of tools
+and libraries for web browser automation. Its [sub-projects] are
+Selenium WebDriver, Selenium Grid, Selenium IDE and Selenium RC.
+Selenium WebDriver implements the W3C WebDriver specification draft
+which describes an API for the interaction with different web
+browsers, such as Chrome or Firefox. Selenium Grid supports the
+execution of tests scripts in a network of remote test nodes.
+The Selenium IDE is a Firefox plug-in which provides a capture
+and replay function. Selenium Remote Control is the predecessor
+of Selenium Grid.
+
+The most important part for us is the Selenium WebDriver. In order
+to get a basic idea of how it is working, you can have a look on the
+following simplified test script.
+
+    [...]
```

Comment:
To highlight text, check the file "examples.md" in epages-docs. There are some examples for highlighting options.